tests/helpers.py

```diff
@@ -1352,3 +1367,53 @@ def get_memory_leak_median_value():
             raise UnexpectedBehaviour
         median_dict[f"{worker}"] = statistics.median(memory_leak_data)
     return median_dict
+
+
+def refresh_oc_login_connection(user=None, password=None):
+    """
+    Function to refresh oc user login
+    Default login using kubeadmin user and password
+
+    Args:
+        user (str): Username to login
+        password (str): Password to login
+    """
+    user = user or config.RUN['username']
+    if not password:
+        filename = os.path.join(
+            config.ENV_DATA['cluster_path'],
+            config.RUN['password_location']
+        )
+        with open(filename) as f:
+            password = f.read()
+    ocs_obj = ocp.OCP()
+    ocs_obj.login(user=user, password=password)
+
+
+def rsync_kubeconf_to_node(node):
+    """
+    Function to copy kubeconfig to OCP node
+
+    Args:
+        node (str): OCP node to copy kubeconfig if not present
+    """
+    # ocp_obj = ocp.OCP()
+    filename = os.path.join(
+        config.ENV_DATA['cluster_path'],
+        config.RUN['kubeconfig_location']
+    )
+    str_split = filename.split('/')
+    str_split = str_split[:-1]
+    file_path = '/'.join(str_split)
```

Comment:
Are you trying to achieve here something which os.path.dirname function already provides to you? :)

Reply:
default filename provides file path '/home/rperiyas/cluster/auth/kubeconfig' , this can't be used in rsync so to remove the kubeconfig. splitting the string using '/' and removing last list and joining again using '/', so that rsync can sync the directory.

Reply:
I understand but look:  
``` python
os.path.dirname("/home/rperiyas/cluster/auth/kubeconfig")
# OUTPUT VALUE:
'/home/rperiyas/cluster/auth'
```
This is what you want no?

Reply:
yes